### ISSUE TITLE
In case of build failure of a demo, there should be a way to see logs

### ISSUE THREAD

**Issue by caldav** [canonical-webteam/demoservice#10]
In case of build failure of a demo, there should be a way to access related logs.

**Comment by tbille** [comment]
Agree. This should be considered in the new structure @jpmartinspt

**Comment by deadlight** [comment]
We should consider using the suggestion in #20 to display the logs in a `<details>` tag
